src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -61,7 +61,7 @@ public LockableResourcesManager() {
 		return matching;
 	}
 
-	public List<LockableResource> getResourcesFromBuild(AbstractBuild<?, ?> build) {
+	public List<LockableResource> getResourcesFromBuild(Run<?, ?> build) {
```

Comment:
@amuniz I know this is changing the API here and you did deliberately keep them separate, but given that this doesn't have side effects and works exactly the same whether you pass it an `AbstractBuild` or a `Run`, I'm not sure I see the reason to have it just be for `AbstractBuild`s, especially since the only places it's called are in `LockRunListener#onCompleted` and `LockRunListener#onDeleted`, which now do need to operate on `Run`s...